package.json

```diff
@@ -33,6 +33,7 @@
     "@testing-library/react-native": "^12.8.1",
     "expo": "^52.0.9",
     "expo-auth-session": "~6.0.0",
+    "expo-blur": "^14.0.1",
```

Comment:
I understand this was to blur the challenges, but it is not used, it would be better to remove it from there.

Reply:
you're right thanks !